### DISCUSSION TITLE
2011/05/25/%E4%BA%94%E4%B8%AA%E5%8D%8A%E6%9C%88%E5%A4%A7%E7%9A%84%E8%B1%86%E8%B1%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#329] [Announcements]
# 2011/05/25/%E4%BA%94%E4%B8%AA%E5%8D%8A%E6%9C%88%E5%A4%A7%E7%9A%84%E8%B1%86%E8%B1%86

学会倒着爬了。豆豆自从学会了翻身，就一直努力往前爬，无奈年纪太小，腿脚根本不听使唤。可怜豆豆一把子力气，不知道往那里用才好。豆豆常常把手拼命往床上撑，胸脯挺得老高，自己跟自己较劲，一会脸就憋的通红了。后来豆豆就不用蛮力了，反正不好使。他有时候会突然一松手，身子扑到在床上，然后再撑起来看看挪了地方没。两个星期前，豆豆学会了原地打转：把肚子作为支点，两只手交错向左或右倒腾，几下子就能来个180度大转弯。最近，豆豆又学会了后退了：两只手先缩到胸前，然后他身体的前上方用力一推，他的肚皮就会朝脚的方向一划。他这一招效率还挺高，我们全家不得不对他提高了安全警戒级别，生怕他自己促溜一下掉到床下去。

https://blog.qizhen.xyz/2011/05/25/%E4%BA%94%E4%B8%AA%E5%8D%8A%E6%9C%88%E5%A4%A7%E7%9A%84%E8%B1%86%E8%B1%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

ronin
2011年06月18日 19:54 编辑
哈哈，孩子像爸爸

回复
carol_su
2011年06月1日 19:45 编辑
哈哈，流哈喇子的小盆友好可爱~~~

回复
